examples/oneapi/dpc/source/decision_forest/df_cls_dense_batch.cpp

```diff
@@ -33,7 +33,7 @@ void run(sycl::queue &queue) {
   const auto x_train = dal::read<dal::table>(queue, dal::csv::data_source{train_data_file_name});
   const auto y_train = dal::read<dal::table>(queue, dal::csv::data_source{train_label_file_name});
 
-  const auto x_test = dal::read<dal::table>(dal::csv::data_source{test_data_file_name});
+  const auto x_test = dal::read<dal::table>(queue, dal::csv::data_source{test_data_file_name});
   const auto y_test = dal::read<dal::table>(dal::csv::data_source{test_label_file_name});
```

Comment:
queue?

Reply:
y_test is used for displaying expected values, therefore no need for queue for it